src/cloudai/_core/installables/base.py

```
@@ -14,9 +14,39 @@
+
+
+class Installable(ABC):
+    @abstractmethod
+    def __eq__(self, other: object) -> bool: ...
+
+    @abstractmethod
+    def __hash__(self) -> int: ...
+
+    def _unsupported_result(self, operation: str) -> "InstallStatusResult":
+        return InstallStatusResult(
+            False,
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)